buidl/psbt_helper.py

```diff
@@ -0,0 +1,199 @@
+from collections import defaultdict
```

Comment:
After some discussion with Chris, I think we have a workaround and we can probably remove this entire file/method from buidl. 

I think that will be *much* cleaner from the build side!